src/tiny_websockets/network/generic_esp/generic_esp_clients.hpp

```diff
@@ -3,6 +3,9 @@
 #include <tiny_websockets/internals/ws_common.hpp>
 #include <tiny_websockets/network/tcp_client.hpp>
 
+// Timeout for connection attempts in milliseconds
+#define readLineTimeout 1000
```

Comment:
Keep the CONSTANTS_IN_CAPITAL_CASE convention 
Also, put it in [ws_config_defs.hpp](https://github.com/gilmaimon/ArduinoWebsockets/blob/master/src/tiny_websockets/ws_config_defs.hpp)